### PR TITLE
add link to IPFS GitHub explanation

### LINKED ISSUE #86: Missing a link to performance issues of using IPFS
A more detailed analysis of performance and functionality issues of IPFS is mentioned here but not linked.

https://docs.radicle.xyz/docs/understanding-radicle/why-radicle#git-gossips-well

### PR DESCRIPTION
**Pull request by brandonhaslegs**
Add missing link to Kim's post on IPFS.

closes #86 

image

### CODE REVIEW

File: docs/understanding-radicle/why-radicle.md [hunk 1]
```diff
@@ -77,35 +78,35 @@ As we set out to build an alternative, we started by thinking about the values
...
 
-4. It must be offline-first
+4.  It must be offline-first
 
     It must not require internet connectivity, DNS or online portals to
     function. There must be no single point of failure and it must be always
     available.
 
-5. It must not compromise on security
+5.  It must not compromise on security
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:107]:
```suggestion
5. It must not compromise on security
```

File: docs/understanding-radicle/why-radicle.md [hunk 2]
```diff
@@ -77,35 +78,35 @@ As we set out to build an alternative, we started by thinking about the values
...
 
-3. It must be user-friendly
+3.  It must be user-friendly
 
     The software must be easy to use and not expect tremendous change in
     behavior from the user. Responsiveness and functionality must meet the
     standards established by current platforms.
 
-4. It must be offline-first
+4.  It must be offline-first
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:101]:
```suggestion
4. It must be offline-first
```

File: docs/understanding-radicle/why-radicle.md [hunk 3]
```diff
@@ -77,35 +78,35 @@ As we set out to build an alternative, we started by thinking about the values
 that we recognize as integral to free and open source code collaboration. With
 that said, we developed the following list of guiding principles:
 
-1. It must prioritize user freedom In the words of the free software movement:
+1.  It must prioritize user freedom In the words of the free software movement:
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:81]:
```suggestion
1. It must prioritize user freedom In the words of the free software movement:
```

File: docs/understanding-radicle/why-radicle.md [hunk 4]
```diff
@@ -77,35 +78,35 @@ As we set out to build an alternative, we started by thinking about the values
...
 
         […] users have the freedom to run, copy, distribute, study, change and
         improve the software. Thus, "free software" is a matter of liberty, not
         price.
 
-2. It must be accessible and uncensorable
+2.  It must be accessible and uncensorable
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:87]:
```suggestion
2. It must be accessible and uncensorable
```

File: docs/understanding-radicle/why-radicle.md [hunk 5]
```diff
@@ -77,35 +78,35 @@ As we set out to build an alternative, we started by thinking about the values
...
 
     Anyone should have the freedom to use the software to collaborate with
     others. No single party should be able to ban users from accessing the
     system, or content from being shared. It must be auditable and transparent.
     In addition, users should have the freedom to control their interactions and
     the content they see on an individual basis.
 
-3. It must be user-friendly
+3.  It must be user-friendly
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:95]:
```suggestion
3. It must be user-friendly
```

File: docs/understanding-radicle/why-radicle.md [hunk 6]
```diff
@@ -181,7 +182,7 @@ data.
 The next design decision came as a result of our experimentation with
 decentralized storage. After building the first version of Radicle on IPFS, we
 ran into performance and functionality issues. A more detailed analysis can be
-found here, but the major realization was that replicating git repos
+found [here](https://github.com/radicle-dev/radicle-alpha/issues/689), but the major realization was that replicating git repos
 peer-to-peer on the storage layer left us no choice but losing the packfile
 protocol, one of the things that makes git fast. This approach would make source
 code a second-class citizen—making it impractical to store repositories with
```
Comment (xla) [inline docs/understanding-radicle/why-radicle.md:188]:
```suggestion
found [here](https://github.com/radicle-dev/radicle-alpha/issues/689), but the
major realization was that replicating git repos peer-to-peer on the storage
layer left us no choice but losing the packfile protocol, one of the things that
makes git fast. This approach would make source code a second-class
citizen—making it impractical to store repositories with
```

**Review by xla — CHANGES_REQUESTED** [review]
Rogue spaces. :space_invader:

**Review by abbey-titcomb — APPROVED** [review]

**Review by rudolfs — APPROVED** [review]

**Pull request merged**